core/code/portal_detail_display.js

```diff
@@ -9,8 +9,8 @@ window.resetScrollOnNewPortal = function() {
   }
 };
 
-window.renderPortalDetails = function(guid) {
-  selectPortal(window.portals[guid] ? guid : null);
+window.renderPortalDetails = function(guid, dontSelect) {
```

Comment:
```suggestion
window.renderPortalDetails = function(guid) {
```
I haven't reviewed this PR as a whole, but still believe that here we need slightly different logic:
**do not select portal if it is already set as current** without additional arguments.

In case if we will found that it is still required in some special case, then we better add opposite:
```suggestion
window.renderPortalDetails = function(guid, forceSelect) {
```

Reply:
Second looks saner.
My though was to not break (unknonwn) plugins that would assume renderPortalDetails triggers a `portalSelected` event as long as the portal has its own marker.
But I bet plugins relying on this will eventually update to match the needs

Reply:
> But I bet plugins relying on this will eventually update to match the needs

So am I.
Let's try to remove `dontSelect`, may be taht will be best for all.